go.mod

```diff
@@ -51,7 +51,7 @@ require (
 	github.com/ssor/bom v0.0.0-20170718123548-6386211fdfcf // indirect
 	github.com/stretchr/testify v1.8.1
 	github.com/wtfutil/spotigopher v0.0.0-20191127141047-7d8168fe103a
-	github.com/wtfutil/todoist v0.0.2-0.20191216004217-0ec29ceda61a
```

Comment:
Note that this (and go.sum) will need updating when the library version is released.